docs/index.md

```diff
@@ -0,0 +1,1488 @@
+---
+hide:
+    - navigation
+---
+
+
+
+#  **A Living Review of Machine Learning for Particle Physics**
+
+*Modern machine learning techniques, including deep learning, is rapidly being applied, adapted, and developed for high energy physics.  The goal of this document is to provide a nearly comprehensive list of citations for those developing and applying these approaches to experimental, phenomenological, or theoretical analyses.  As a living document, it will be updated as often as possible to incorporate the latest developments.  A list of proper (unchanging) reviews can be found within.  Papers are grouped into a small set of topics to be as useful as possible.  Suggestions are most welcome.*
+
+[![download](https://img.shields.io/badge/download-review-blue.svg)](https://iml-wg.github.io/HEPML-LivingReview/review/hepml-review.pdf)
+[![github](https://badges.aleen42.com/src/github.svg)](https://github.com/iml-wg/HEPML-LivingReview)
+
+## Reviews
+
+??? example "Modern reviews"
+    <div class="meta_for_parser tablespecs" style="font-size: 1pt;visibility:hidden" markdown>
+    ### Modern reviews
+    </div>
```

Comment:
These changes are needed in the CI as well as general formatting:

- three question marks for collapsable section
- div element to hide the section header (and set font size small for spacing reasons) so that TOC still works at this depth, but not visibly duplicating the title which is already in the toggle box
- Content of box needs four space indent